### PR TITLE
WRO-11352: CustomSkin generator: Fix custom skin components colors.

### PR DESCRIPTION
**Pull request by alexandrumorariu**
Enact-DCO-1.0-Signed-off-by: Alexandru Morariu alexandru.morariu@lgepartner.com

### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Changed the variables that overwrite the skin ones to cover the correct css attributes.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-11352

### CODE REVIEW

File: sandstone/custom-skin/src/components/OutputField/OutputField.js [hunk 1]
```diff
@@ -86,7 +91,7 @@ const OutputField = kind({
...
 					{!platform.webos ? <TooltipButton className={css.outputBtn} css={css} icon="files" minWidth={false} onBlur={handleClose} onClick={copyToClipboard} onFocus={handleFocus} size="small" tooltipText="Copy to clipboard">Copy</TooltipButton> : ''}
 					{!platform.webos ? <TooltipButton className={css.outputBtn} css={css} icon="download" minWidth={false} onBlur={handleClose} onClick={generateFile} onFocus={handleFocus} size="small" tooltipText="Get CSS file">Download</TooltipButton> : ''}
-					<TooltipButton className={css.outputBtn} css={css} icon="refresh" minWidth={false} onBlur={handleClose} onClick={setDefaultState} onFocus={handleFocus} size="small" tooltipText="Restore skin to default colors">Reset</TooltipButton> {/* eslint-disable-line */}
+					<TooltipButton open className={css.outputBtn} css={css} icon="refresh" minWidth={false} onBlur={handleClose} onClick={setDefaultState} onFocus={handleFocus} size="small" tooltipText="Restore skin to default colors">Reset</TooltipButton> {/* eslint-disable-line */}
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/OutputField/OutputField.js:94]: Please sort props.

**Comment by stanca-pop-lgp** [comment]
Placeholder in Skin Name input still changes when we change component text color.
image

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Pull request merged**